### PR TITLE
Fix/first quest message flow

### PR DESCRIPTION
**Pull request by tikisailor**
Prevent FQ from being triggered twice when invoking /start command (due to transition to event trigger)

### CODE REVIEW

**Review by amaredeus — CHANGES_REQUESTED** [review]
Was thinking about this.. to reduce the complexity and make it super simple. Let's have the first quest start command do two things

1. Removes all first quest related roles
2. Add the first quest start verification role

That should trigger the flow automatically instead of manually trigger

**Review by nonsensecodes — DISMISSED** [review]
That was a simple cut. Thank you, @tikisailor! But also, looks like @amaredeus has some suggestions 👍

**Comment by amaredeus** [comment]
Looks good, let's update the changelog and we should be good with this PR!

**Comment by tikisailor** [comment]
> Looks good, let's update the changelog and we should be good with this PR!

Ok, please check if this is right. I put the date for release sched on tuesday

**Comment by tikisailor** [comment]
> Looks good, let's update the changelog and we should be good with this PR!

Ok, please check if this is right. I put the date for release sched on tuesday

**Review by amaredeus — APPROVED** [review]

**Review by amaredeus — APPROVED** [review]

File: src/app/service/first-quest/LaunchFirstQuest.ts [hunk 1]
```diff
@@ -67,10 +67,8 @@ export const sendFqMessage = async (dmChan:TextBasedChannels | string, member: G
 		if (firstQuestMessage.id === dmChannel.lastMessage.id) {
 			try {
 				await dmChannel.send('The conversation timed out. ' +
-				'All your progress has been saved. ' +
-				'You can continue at any time by ' +
-				'responding to this conversation ' +
-				'with **!first-quest** ');
+				'You can restart First Quest ' +
+				'using the **/first-quest start** command');
```
Comment (nonsensecodes) [inline src/app/service/first-quest/LaunchFirstQuest.ts:71]: This makes more sense. Awesome.

**Review by nonsensecodes — APPROVED** [review]
Looks good! Makes sense to switch from the bang in DMs to slash-commands in the channel. Great job, @tikisailor 🔥

**Pull request merged**